packages/services/api/src/modules/support/module.graphql.ts

```diff
@@ -30,6 +30,9 @@ export default gql`
 
   input SupportTicketCreateInput {
     organizationSlug: String!
+    project: String
+    target: String
+    category: SupportCategory!
```

Comment:
Minor: I defer to my colleagues, but I think we should default this or make it nullable to avoid taking any downtime during the deployment window.

We don't get a ton of support tickets so I doubt it'd cause an issue, but I'm extra cautious.